unpackaged/functional_tests/address/classes/Address_FTST.cls

```diff
@@ -290,4 +310,114 @@ public with sharing class Address_FTST {
             'The current address is incorrect'
         );
     }
+
+    /**
+     * @description T-3904085: Using Household account model, modify an existing Contact Address and assess new address
+     * creation under Household account
+     */
+    @isTest
+    private static void updateExistingHouseholdAccountContactAddress() {
+        hed__Hierarchy_Settings__c orgSettings = createSettings();
```

Comment:
I notice that this settings setup approach has been merged already but we do have some settings pattern in eda package. My understand is that you don't have to upsert the settings as it is static.

Reply:
you are right, the upsert and update of orgSettings variable is not really needed. Just fixed it. Thanks!